src/components/crisis-overview/crisis-overview.js

```diff
@@ -30,6 +31,22 @@ CrisisOverviewWidget.prototype.compile = function(elements, next) {
   var config = this.config();
   this.config('adjustedTitle', titleAdjust(config.title));
 
+  // Traverses data indicators looking for points that are structured as API-driven content.
+  // This is then formatted via our custom 'en-long' language to handle quantifier mappings.
+  numeral.language('en-long');
+  for (var i in config.indicators) {
+    for (var j in config.indicators[i].data) {
+      var original = config.indicators[i].data[j];
+      if (original.figure != null && typeof original.figure === 'object' && original.figure.type == 'request') {
+        var value = numeral(original.figure.content[0].value).format('0.00 a').split(' ');
+        this.config('indicators.' + i + '.data.' + j + '.figure', value[0]);
+        if (!original.quantifier) {
+          this.config('indicators.' + i + '.data.' + 0 + '.quantifier', value[1]);
+        }
+      }
+    }
+  }
+
```

Comment:
This section is the main change of the PR. It switches from the default, built-in `en` language to our custom `en-long`. It then scans through all the indicators in the stats box and looks for items that are API-driven to be reformatted.

If the quantifier is not already set, it will insert it programmatically based on the formatting of the numeral library.